models/classes/media/LocalItemSource.php

```diff
@@ -314,4 +314,9 @@ protected function getItemDirectory()
         }
         return $this->itemService->getItemDirectory($this->item, $this->lang);
     }
+
+    public function getProcessedFileStream(string $link): StreamInterface
```

Comment:
IMO You do not need this method: https://github.com/oat-sa/tao-core/pull/2551/files#r441299754